File: examples/event/read/go/readEventsOnATopic.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/event"
)

// Read stored events
func ReadEventsOnAtopic() {
	eventService := event.NewEventService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := eventService.Read(&event.ReadRequest{
		Topic: "user",
	})
	fmt.Println(rsp, err)
}

File: examples/event/read/node/readEventsOnATopic.js
const { EventService } = require("micro-js-client/event");

// Read stored events
async function readEventsOnAtopic() {
  let eventService = new EventService(process.env.MICRO_API_TOKEN);
  let rsp = await eventService.read({
    topic: "user",
  });
  console.log(rsp);
}

readEventsOnAtopic();
